src/main/java/org/gridsuite/sensitivityanalysis/server/repositories/SensitivityAnalysisResultRepository.java

```diff
@@ -124,8 +130,12 @@ public void insert(UUID resultUuid, SensitivityAnalysisResult result, String sta
     @Transactional
     public void delete(UUID resultUuid) {
         Objects.requireNonNull(resultUuid);
+        AtomicReference<Long> startTime = new AtomicReference<>();
+        startTime.set(System.nanoTime());
+        sensitivityRepository.deleteSensitivityBySensitivityAnalysisResultUUid(resultUuid);
```

Comment:
Just for homogeneity with other services delete globalstatus first then sensitivities then finally the result